src/test/java/uk/gov/hmcts/reform/sscs/personalisation/PersonalisationTest.java

```diff
@@ -139,12 +148,50 @@ public void setup() {
         name = Name.builder().firstName("Harry").lastName("Kane").title("Mr").build();
     }
 
+    @Test
+    @Parameters(method = "generateNotificationTypeAndSubscriptionsScenarios")
+    public void givenSubscriptionType_shouldGenerateEmailAndSmsTemplateNamesPerSubscription(
+            NotificationEventType notificationEventType, HearingType hearingType) {
+        NotificationWrapper notificationWrapper = new CcdNotificationWrapper(SscsCaseDataWrapper.builder()
+                .newSscsCaseData(SscsCaseData.builder()
+                        .appeal(Appeal.builder()
+                                .hearingType(hearingType.name())
+                                .build())
+                        .build())
+                .notificationEventType(notificationEventType)
+                .build());
+
+        personalisation.getTemplate(notificationWrapper, PIP, null);
+
+        verify(config).getTemplate(eq(getExpectedTemplateName(notificationEventType, null)),
+                anyString(), any(Benefit.class), any(AppealHearingType.class));
+    }
+
+    private String getExpectedTemplateName(NotificationEventType notificationEventType,
+                                           SubscriptionType subscriptionType) {
+        return notificationEventType.getId() + (subscriptionType == null ? "" :
+                "." + subscriptionType.name().toLowerCase());
+    }
+
+    @SuppressWarnings("Indentation")
+    private Object[] generateNotificationTypeAndSubscriptionsScenarios() {
+        return new Object[]{
+                new Object[]{SYA_APPEAL_CREATED_NOTIFICATION, PAPER},
```

Comment:
Change the eventType to AppealLapsed since that's the event this ticket is about, please.